### DISCUSSION TITLE
Fysect ADXL - Axes definition

### DISCUSSION THREAD

**Discussion by dev-zetta** [Guilouz/Klipper-Flsun-Speeder-Pad#116] [Q&A] [answered]
Can you please tell me whenever the axes map definition is the same here - I have the ADXL mounted on the fan.

axes_map: x,-z,y

![IMG20230704171857](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/3283719/9dda7388-5ce2-4c86-879b-43fcff1124af)

**Comment by Guilouz** [comment, marked as answer]
Yes your axis map is correct.
